README_MACOS.md

```diff
@@ -36,22 +36,18 @@ Inside that application's folder (`SuperCollider.app/Contents/MacOS/`) are the t
 Prerequisites:
 -------------
 
-- **Xcode** can be installed free from the Apple App Store or downloaded from: http://developer.apple.com
-
-  - Xcode 5 may work
-  - Xcode 6 is known to work - it requires a Mac running macOS version 10.9.4 or later or 10.10
-  - Later versions should definitely work
-
-- **Xcode command line tools** must be installed - after installing Xcode, this can be done from the Xcode preferences or from the command line:
+- **Xcode** can be installed free from the Apple App Store or downloaded from: http://developer.apple.com.
+  Xcode >= 7 is recommended. Xcode 6 will work, but requires macOS >= 10.9.4.
+- If you don't have **Xcode command line tools** already, install them with:
```

Comment:
don't >> do not

Reply:
can you explain a little more why you want that change? i was just following the existing tone of the readmes, which is fairly casual. there's a usage of "don't" below (line 95) and other contractions sprinkled throughout. i'd be happy to have a discussion about it, just want to know where you're coming from.

Reply:
it was just irritating to read... but merely a suggestion (hence I "approved" rather than "requestewd changes") :)
The other contractions just slipped my mind... I'm fine with either.

Reply:
alright, changed to "do not" and squashed with previous commit. i agree that it reads better without the contraction